jetty-documentation/src/main/asciidoc/quick-start/introduction/what-version.adoc

```diff
@@ -33,9 +33,12 @@ _____
 [width="100%",cols="12%,9%,15%,6%,21%,10%,6%,21%",options="header",]
 |=======================================================================
 |Version |Year |Home |JVM |Protocols |Servlet |JSP |Status
-|9.4 |2016- |Eclipse |1.8 |HTTP/1.1 (RFC 7230), HTTP/2 (RFC 7540), WebSocket (RFC 6455, JSR 356), FastCGI |3.1 |2.3 |Stable
+|10  |2019- |Eclipse |11- |HTTP/1.1 (RFC 7230), HTTP/2 (RFC 7540), WebSocket (RFC 6455, JSR 356), FastCGI |4.0.2 |2.3 |*UNSTABLE / Alpha*
+|9.4 |2016- |Eclipse |1.8-12.0.1 |HTTP/1.1 (RFC 7230), HTTP/2 (RFC 7540), WebSocket (RFC 6455, JSR 356), FastCGI |3.1 |2.3 |Stable
```

Comment:
This should be `1.8-` only to mean at least 1.8 but up to current, no? Like above Jetty 10 is `11-`, right?

Reply:
Agreed with Simone. 9.4 should be 1.8- until we decide to cap it at a certain version. Generally, I've looked at that column as a standard for what the minimum version is rather than inclusion of all versions it runs on.

Reply:
I'm a bit torn personally. (you can see I used both forms in this PR)

The use of `<ver>-` is very programmer-speak and isn't very user friendly or appropriate for a general purpose documentation like this.

I'm more and more in favor of keeping the list up to date, and use the form `<min-jdk-ver>-<max-tested-jdk-ver>` range.

But I could be talked into `<min-jdk-ver> and newer` aka `1.8 and newer` as well.

Reply:
I had a brief moment of thought to just rename the column "JVM" to "Minimum JVM".
But that doesn't work for version of Jetty prior to 9.4 and Java 9+ (think MultiRelease JARs, bytecode scanning, asm versions, and Classloader shenanigans)

Reply:
I think "Minimum JVM" column is the right way to go, with an asterisk if features are available in newer JVM versions.

So to me Jetty 9.2.x is "Minimum JDK = JDK 5[*]" with the [*] meaning "Some feature available in newer JVMs" (e.g. IIRC: SNI and SPDY).

Reply:
+1. We can just add a note below the table that says something along the lines of, "Minimum version of Java required in this release branch. Some later releases may require a newer JVM for additional features."

Reply:
I updated the table.  The column is now "Min JVM" and the versions can have footnotes.

Btw, I tried to use [asciidoc footnotes](https://www.methods.co.nz/asciidoc/asciidoc.css-embedded.html#X92), but our jetty-documentation setup didn't like it.

It now renders like this ...

![jetty-versions](https://user-images.githubusercontent.com/191721/64433676-907ecb80-d084-11e9-99c7-ead1e3e2e3e3.png)